src/game/Cbuf/Cbuf.cpp

```diff
@@ -53,4 +53,38 @@ int Conbuf_CleanText(const char* source, char* target, int sizeofTarget)
 
 	*target = 0;
 	return target - start;
+}
+
+//THUNK : 0x004F5770
```

Comment:
thunk?

Reply:
Still waiting on clarification on when to use those.